### PR TITLE
Pull request for mobile first

### PR DESCRIPTION
**Pull request by codehass**
In this Pull request :
- Add header as a container of the page.
- Add the button effect.
- Add the hamburger button 
- Fix image paths.

 Thank you for reviewing my Pull request.

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @codehass, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

   - You have submitted a project that has failing linters, Kindly make sure your linters are all passing before submitting the PR.

## Stlylint AutoFix
- [ ] if you have a lot of CSS line errors, I kindly suggest you fix them by running `npx stylelint --fix "**/*.{css,scss}"` on your command line, but please get used to fixing the errors line by line also because it'll help you learn best practices.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
```
Comment (nedjwab) [inline index.html]: [ ] The container of the section should be a `header` tag. Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info. kindly fix that 🤓

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <nav>
+        <img src="/imgs/image_geometry_1.png" alt="image" />
```
Comment (nedjwab) [inline index.html]: [ ] kindly add the hamburger button to your design, it's part of the requirements.🤓

The Figma design 👇 :
hamburgerdesign


Your design 👇 : 
hamburgerta3ou

File: style.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+@media (max-width: 767px) {
+  .container {
+    width: 100%;
+    height: 74px;
+    height: 568px;
+  }
+
+  nav {
+    width: 100%;
+    display: flex;
```
Comment (nedjwab) [inline style.css:15]: Nice use of the flex 🔥🔥🔥🔥

File: style.css [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    height: 126px;
+    font-family: "Poppins", sans-serif;
+    font-size: 14px;
+    line-height: 21px;
+    font-weight: 400;
+    letter-spacing: -1%;
+    margin: 0;
+  }
+
+  .headline-cont button {
```
Comment (nedjwab) [inline style.css:54]: [ ] kindly add the hover effect to your button. they are part of the requirements kindly fix that 🤓.
The Figma design👇 :
buttonhover

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @codehass  

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
check the comments below 👇👇👇👇👇
 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <img src="/imgs/image_geometry_1.png" alt="image" />
+        <img src="/imgs/ic_menu (1).png" alt="Menu" class="menu-icon" />
```
Comment (Nyame-Wolf) [inline index.html]: [OPTIONAL] Kindly note that this is supposed to be a background image. The previous reviewer might not have highlighted it but please change it as its part of the requirements

Comment (Nyame-Wolf) [inline index.html]: Please fix the paths to your images. Remove the stroke before the path i.e: from 
`Menu` to Menu
This is the current display of your page:
Screenshot 2022-10-27 at 16 49 33

File: index.html [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+          <p>
+            Hello! I am a software developer! I can help you build a product,
+            feature or website. Taka a look of my works. If you like what you
+            see and have a project you need coded, don’t hesitate and contact
+            me.
+          </p>
+          <button>Start collaboration</button>
+        </div>
+      </section>
+      <img src="/imgs/image_geometry_2.png" class="headline-img" />
```
Comment (Nyame-Wolf) [inline index.html]: [OPTIONAL] Kindly note that this is supposed to be a background image. The previous reviewer might not have highlighted it but please change it as its part of the requirements

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @codehass  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Linters are passing
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by sajid-munawar — APPROVED** [review]
### Hi @codehass , 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
 ## [Optional] suggestions:
 * [ ]  Please check my inline comments.

> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @sajid-munawar in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**